MonoGame.Framework/Content/ContentManager.cs

```diff
@@ -668,5 +669,13 @@ public IServiceProvider ServiceProvider
 				return this.serviceProvider;
 			}
 		}
+
+        internal byte[] GetScratchBuffer(int size)
```

Comment:
Eventually we could consider making this public for those writing custom content readers.  We would have to add docs to it explaining its usage and benefits.